app/Http/Controllers/SupportController.php

```diff
@@ -10,10 +10,27 @@
 use App\Models\Camp;
 use App\Models\Topic;
```

Comment:
We can remove `AddNickNameRequest` and `UpdateNickNameRequest` from the Imports.